### DISCUSSION TITLE
DocTr is not recognizing german characters like "öäü"

### DISCUSSION THREAD

**Discussion by Moorky** [mindee/doctr#1317] [Q&A]
In the documentation it says that the models support german language, but when I try extracting words from different german documents "ÖÄÜ" gets changed to "OAU".

Am I doing something wrong?

I'm using
    DOCTR_WORD:
      PT: doctr/db_resnet50/pt/db_resnet50-ac60cadc.pt
    DOCTR_RECOGNITION:
      PT: doctr/crnn_vgg16_bn/pt/crnn_vgg16_bn-9762b0b0.pt

**Comment by felixT2K** [comment]
Hi @Moorky 👋🏼,

docTR's recognition models are only pretrained on french vocab :)
Where does it say that it's able to extract german letters ? :)
Do you mean the vocab overview ? That's only a list of predefined vocabs you can use to fine tune your own model
But if you have some data you can fine tune your own model: https://mindee.github.io/doctr/using_doctr/custom_models_training.html
